### PR TITLE
docs: Improve the Working with Chrome Extensions section

### PR DESCRIPTION
**Pull request by kzar**
- Added a note to mention the experimental Chrome headless mode can
  also be used.
- Improved the example code to wait for the background page to be
  ready. The previous example sometimes didn't work, since the
  background page would take a split-second to load.
- Added a note to mention how to find the background ServiceWorker for
  Chrome MV3 extensions.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Comment by kzar** [comment]
Thanks for the quick review @OrKoN ! I clicked to rebase this branch, but changes are landing too quickly for the CI to complete before another rebase is required. Do you need me to keep rebasing (or to do anything else), or can I leave this to you guys to merge?

**Comment by OrKoN** [comment]
For some reason GitHub does not allow me to rebase the PR and it just says that the branch is out of date which will prevent merging. Perhaps you could give us the permissions to update the branch (then I can rebase to whenever needed)? I think it's the doc about the feature https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by kzar** [comment]
Cool OK I've checked "Allow edits by maintainers", does it work for you now?

**Comment by OrKoN** [comment]
thanks, it did work!

**Comment by kzar** [comment]
Nice, thanks :+1:

**Pull request merged**
